Common/triggerSel.py

```diff
@@ -40,7 +50,7 @@ def ApplyTriggers(self, df, isData = False):
             df = df.Define(f'hasOOMatching_{path}', f'HasOOMatching({legVector} )')
             fullPathSelection = f'{or_paths} &&  hasOOMatching_{path}'
             fullPathSelection += ' && '.join(additional_conditions)
-            hltBranch = f'HLT_{path}'
+            hltBranch = f'HLT_{trigName}'
```

Comment:
should be `path` as for the whole block, not for each component.

Reply:
ok

Reply:
ok